assets/js/stats/unique.js

```diff
@@ -0,0 +1,33 @@
+export function updateDailyUnique( key ) {
+	const date = new Date();
+	const expiresAtTimestamp = date.getTime() + 24 * 60 * 60 * 1000;
+	window.localStorage[key] = expiresAtTimestamp;
```

Comment:
We should add a way to clean these up eventually, `localStorage` is for forever.

Reply:
added a possible way to do that, we can refine later.